packages/web-components/src/components/leadspace/leadspace.ts

```diff
@@ -59,20 +59,50 @@ class DDSLeadSpace extends StableSelectorMixin(LitElement) {
     });
   }
 
+  /**
+   * Returns a class-name based on the size parameter type
+   */
+  protected _getContainerClass() {
+    return classMap({
+      [`${prefix}--leadspace__container-medium`]: this.size === LEADSPACE_SIZE.MEDIUM,
```

Comment:
BEM uses two dashes for variants:

```suggestion
      [`${prefix}--leadspace__container--medium`]: this.size === LEADSPACE_SIZE.MEDIUM,
```